GGJSGA23/Assets/MenuManager.cs

```diff
@@ -19,28 +22,29 @@ public class MenuManager : MonoBehaviour
 	[SerializeField] private string threePlayerScene;
 	[SerializeField] private string fourPlayerScene;
 
-	[SerializeField] private string gameOverScene;
+	//[SerializeField] private string gameOverScene;
 
 	//bool changingScene = false;
 
-	// Start is called before the first frame update
+
 	void Start()
 	{
+		if (Instance == null)
+		{
+            Instance = this;
+        }
+        if (Instance != this)
+		{
+			Destroy(gameObject);
+			return;
+		}
+		
```

Comment:
Not sure what happened with intendation in the preview file but I guess it looks fine in the editor?